hazelcast/src/main/java/com/hazelcast/monitor/LocalWanPublisherStats.java

```diff
@@ -80,4 +81,9 @@
      * Returns the last results of the consistency checks, mapped by map name.
      */
     Map<String, ConsistencyCheckResult> getLastConsistencyCheckResults();
+
+    /**
+     * Returns the last synchronization statistics, mapped by map name.
+     */
+    Map<String, WanSyncStats> getLastSyncStats();
```

Comment:
maybe we can enhance javaDoc to tell why only the last sync stats is important?

Reply:
:+1: I'll add this in a separate PR, merging this one for now.